tests/series_only/value_counts_test.py

```diff
@@ -0,0 +1,27 @@
+from typing import Any
+
+import numpy as np
+import pytest
+
+import narwhals.stable.v1 as nw
+
+data = {"a": [4, 4, 6]}
+
+
+def test_value_counts(request: Any, constructor: Any) -> None:
+    if "pandas_series_nullable" in str(constructor):  # fails for py3.8
+        request.applymarker(pytest.mark.xfail)
```

Comment:
I think we may have inadvertently been skipping the rest of the test because of the `pytest.skip` 🙈  taking a look

Reply:
![image](https://github.com/user-attachments/assets/a75f68ad-259f-4805-ad30-ef2b964d8779)
I thought that a dot means a passed test, and "s" a skipped one. But it makes sense Anyway I'll take care of sys.version_info

Reply:
is this on your branch? I think you're doing it correctly, but on `main`, maybe not?

Reply:
I'm not sure if I understand correctly. But yes I did run the test on my branch with changes(arrow-valcounts)
![image](https://github.com/user-attachments/assets/9a36ac2f-be45-485f-aae6-18e204956c9c)
when I switch to main I cannot run the test, because it doesn't exist

Reply:
sorry I meant, the corresponding test:
```
pytest tests/series_only/test_common.py -k value_counts
```
still, looks like only one test gets skipped, so we're all good 😇